Sources/Component/Label/LabelStyle.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import UIKit
+
+/// A data structure describing the styles of a `Label`.
+public struct LabelStyle {
+    /// The color used to render the text of a `Label`.
+    public let color: UIColor
+
+    /// The font used to render the text of a `Label`.
+    public let font: UIFont
+
+    /// The number of lines the `Label` is allowed to line break.
+    public let numberOfLines: Int
+
+    /// The designated initializer of `LabelStyle`.
+    ///
+    /// - Parameters:
+    ///   - color: The color used to render the text of a `Label`.
+    ///   - font: The font used to render the text of a `Label`.
+    ///   - numberOfLines: The number of lines the `Label` is allowed to line break.
+    public init(
+        color: UIColor,
```

Comment:
I suggest to provide the default values here to make the parameters optional.

Reply:
Yes this was a mistake thanks!